pkg/planner/core/point_get_plan.go

```diff
@@ -315,7 +315,7 @@ type BatchPointGetPlan struct {
 	dbName           string
 	TblInfo          *model.TableInfo
 	IndexInfo        *model.IndexInfo
-	PartitionInfos   []*model.PartitionDefinition
+	PartitionDefs    []*model.PartitionDefinition
```

Comment:
Why name it "Partition"? This is different from PointGet.

Reply:
It's better to use `PartitionDefs` or `Partitions`.

Reply:
I now also changed `PointGetPlan`, to be `partitionDef`, so it is consistent.